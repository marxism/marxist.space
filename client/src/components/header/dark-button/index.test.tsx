/* eslint-env jest */

import * as React from 'react'
import { HeaderDarkButton } from '.'
import { create } from 'react-test-renderer'

describe('HeaderDarkButton', () => {
  test('renders', () => {
    expect(create(<HeaderDarkButton />).toJSON()).toMatchSnapshot()
  })
})
