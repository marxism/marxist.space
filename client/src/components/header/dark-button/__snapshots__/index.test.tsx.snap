// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderDarkButton renders 1`] = `
.c1 {
  width: 20px;
  height: 20px;
}

.c1 path {
  -webkit-transition: fill 0.1s ease;
  transition: fill 0.1s ease;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  padding: 0 16px;
  padding: 0 8px;
  cursor: pointer;
}

@media (max-width:425px) {
  .c1 {
    width: 18px;
    height: 18px;
  }
}

@media (max-width:425px) {
  .c0 {
    padding: 0 8px;
  }
}

@media (hover:hover) {

}

<span
  className="c0"
>
  <svg
    className="c1"
    viewBox="0 0 24 24"
  >
    <path
      d="M6.03569223,7.86020138e-11 C4.77338857,1.342144 4,3.14939605 4,5.13728269 C4,9.27941831 7.35786438,12.6372827 11.5,12.6372827 C13.4878866,12.6372827 15.2951387,11.8638941 16.6372827,10.6015905 C15.5809549,14.0943073 12.3374493,16.6372827 8.5,16.6372827 C3.80557963,16.6372827 0,12.8317031 0,8.13728269 C0,4.29983338 2.54297542,1.05632781 6.03569223,0 Z"
      transform="translate(4 4)"
    />
  </svg>
</span>
`
